RecoLocalCalo/HcalRecAlgos/src/HcalSeverityLevelComputer.cc

```diff
@@ -228,7 +228,11 @@ HcalSeverityLevelComputer::HcalSeverityLevelComputer( const edm::ParameterSet& i
 } // HcalSeverityLevelComputer::HcalSeverityLevelComputer
 
 
-HcalSeverityLevelComputer::~HcalSeverityLevelComputer() {}
+HcalSeverityLevelComputer::~HcalSeverityLevelComputer()
+{
+    delete DropChannel_;
```

Comment:
For the future, you might want to consider fixing memory problems by changing the type to be a `std::unique_ptr<>` since that will handle not only memory cleanup but also properly handle copy constructors and `operator=`. In general, using bare pointers for memory management should be something we probably want to avoid in CMSSW. [Although just letting memory leak is worse than not using a smart pointer ;)]